docs/src/templates/includes/documentation-components/subnavs/subnav--sidebar.hbs

```diff
@@ -160,9 +160,52 @@
 
 {{#is navsub 'docs-css'}}
 <!-- subnav for Documentation > CSS -->
-    <li><a href="{{ baseUrl }}/documentation/css/" class="{{#is navactive 'css-principles'}} is-active{{/is}}">Principles</a></li>
-    <li><a href="{{ baseUrl }}/documentation/css/css-naming" class="{{#is navactive 'css-naming'}} is-active{{/is}}">Naming</a></li>
-    <li><a href="{{ baseUrl }}/documentation/css/structure" class="{{#is navactive 'css-structure'}} is-active{{/is}}">Structure</a></li>
+    <li><a href="{{ baseUrl }}/documentation/css/" class="{{#is navactive 'css-principles'}} is-active{{/is}}">Principles</a>
+        {{#is navactive 'css-principles'}}
+        <ul class="sg-sideNav-subnav u-unstyled">
+            <li><a href="#philosophy">Philosophy</a></li>
+            <li><a href="#bestpractices">Best Practices</a></li>
+            <li><a href="#preprocessors">Preprocessors</a></li>
+            <li><a href="#progressive">Progressive Enhancement</a></li>
+        </ul>
+        {{/is}}
+    </li>
+    <li><a href="{{ baseUrl }}/documentation/css/sass" class="{{#is navactive 'sass'}} is-active{{/is}}">Sass</a>
```

Comment:
should all anchor tags be on a new line?

Reply:
It's not something we've discussed in terms of whether we do this or not.

I'm pretty easy either way – as if it's all on one line, looks tidier to me, but can see it from the other side too where people like new tags on new lines.